examples/runners/wgpu/src/compute.rs

```diff
@@ -27,19 +25,19 @@ fn create_device_queue() -> (wgpu::Device, wgpu::Queue) {
     }
     #[cfg(not(target_arch = "wasm32"))]
     {
-        return futures::executor::block_on(create_device_queue_async());
-    };
+        futures::executor::block_on(create_device_queue_async())
+    }
     #[cfg(target_arch = "wasm32")]
     {
-        return wasm_bindgen_futures::spawn_local(create_device_queue_async());
-    };
+        wasm_bindgen_futures::spawn_local(create_device_queue_async())
+    }
```

Comment:
This could be reduced to the following since you're using `cfg_if` already. For some reason I can't provide it as a suggestion on GitHub.

```rust
cfg_if::cfg_if! {
     if #[cfg(target_arch = "wasm32")] {	
        wasm_bindgen_futures::spawn_local(create_device_queue_async())
    } else {
        futures::executor::block_on(create_device_queue_async())
   }
}
```

Reply:
done